src/main/resources/config.yml

```diff
@@ -81,6 +81,13 @@ hacks:
       - SLOW_FALLING
   DebugWand:
     enabled: true
+  DogFacts:
+    enabled: false
+    #In Mins
+    interval: 30
```

Comment:
Actually use this variable instead of only putting it in the config and use the parseTime method for it